packages/translate/src/index.js

```diff
@@ -312,6 +312,9 @@ Translate.prototype.getLanguages = function(target, callback) {
  *     is written in.
  * @param {string} options.to - The ISO 639-1 language code to translate the
  *     input to.
+ * @param {string} options.model - **Note:** Users must be whitelisted to use
```

Comment:
You can add something like "The `model` type requested for this translation. Valid values are listed in public documentation."  We'll add the link ready.